packages/commonwealth/client/scripts/hooks/useWindowResize.ts

```diff
@@ -14,7 +15,9 @@ const useWindowResize = ({ setMenu }) => {
   );
 
   const onWindowResize = useCallback(() => {
-    setMenu({ name: 'default', isVisible: !isWindowSmallInclusive });
+    if (isMobile && window.innerWidth < 425) {
```

Comment:
Why `425`?  We have breakpoints defined in `useBrowserWindow` and `breakpoints.module.scss`. Please use those values